core/data_migration_scripts/4_5_0_stages/EE_DMS_4_5_0_invoice_settings.dmsstage.php

```diff
@@ -75,7 +75,8 @@ protected function _migration_step($num_items = 1)
             // message templates are only initialized AFTER migrations and those two are new in 4.5. So if we wanted to
             // update them from a DMS, we'd need to have the DMS create the message templates which is quite a lot of code;
             // also we don't want to build a dependency on the messages code because it is likely to change soon
-            if (! in_array($invoice_settings['invoice_css'], array( '', 'simple.css' ))) {
+            if (isset($invoice_settings['invoice_css'])
+                && ! in_array($invoice_settings['invoice_css'], array( '', 'simple.css' ))) {
```

Comment:
We can use short array syntax now, so you could do something like

```suggestion
                && ! in_array($invoice_settings['invoice_css'], ['', 'simple.css']))) {
```